#Apply corrections suggested in the head of the generated Makefile
--- a/doc/xml/Makefile.am
+++ b/doc/xml/Makefile.am
@@ -12,11 +12,11 @@
 SUBDIRS= .
 
 EXTRA_DIST =			\
-	docbook.xsl		\
-	pilot-link.docbook
+	../docbook.xsl		\
+	../pilot-link.docbook
 
 all:
-	$(XSLTPROC) $(XSLTPROC_FLAGS) -o $(top_srcdir)/doc/man/ \
+	xsltproc $(XSLTPROC_FLAGS) -o $(top_srcdir)/doc/man/ \
 	$(top_srcdir)/doc/xml/docbook.xsl \
 	$(top_srcdir)/doc/xml/pilot-link.docbook
 
diff --git a/doc/xml/docbook.xsl.in b/doc/xml/docbook.xsl.in
index 85b39439..a3828e81 100644
--- a/doc/xml/docbook.xsl.in
+++ b/doc/xml/docbook.xsl.in
@@ -4,8 +4,8 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 version='1.0'>
 
-<xsl:import href="/usr/share/xml/docbook/stylesheet/docbook-xsl/html/docbook.xsl"/>
-<xsl:include href="/usr/share/xml/docbook/stylesheet/docbook-xsl/manpages/synop.xsl"/>
+<xsl:import href="/usr/share/xml/docbook/xsl-stylesheets/html/docbook.xsl"/>
+<xsl:include href="/usr/share/xml/docbook/xsl-stylesheets/manpages/synop.xsl"/>
 
 <!-- Needed for chunker.xsl (for now): -->
 <xsl:param name="chunker.output.method" select="'text'"/>
